SPECS/rook/rook.spec

```diff
@@ -0,0 +1,1199 @@
+Source99:       update-tarball.sh
+# When possible, a patch is preferred over link-time overrides because the patch will fail if the
+# upstream source is updated without the package maintainers knowing. Patches reduce user error when
+# creating a new SUSE release branch of Rook.
+# Change the default FlexVolume dir path to support Kubic.
+Patch0:         flexvolume-dir.patch
+# Ceph version is needed to set correct container tag in manifests
+BuildRequires:  ceph
+# Rook requirements
+BuildRequires:  curl
+BuildRequires:  git
+BuildRequires:  golang
+# Go and spec requirements
+BuildRequires:  golang-packaging
+BuildRequires:  grep
+BuildRequires:  xz
+# From Ceph base container: github.com/ceph/ceph-container/src/daemon-base/...
+Requires:       patterns-ceph-containers-ceph_base
+# Rook runtime requirements - referenced from packages installed in Rook images
+# From images/ceph/Dockerfile
+Requires:       tini
+
+%description
+Rook is a cloud-native storage orchestrator for Kubernetes, providing
+the platform, framework, and support for a diverse set of storage
+solutions to integrate with cloud-native environments.
+
+See https://github.com/rook/rook for more information.
+
+################################################################################
+# Rook FlexVolume driver metadata
+################################################################################
+%package rookflex
+Summary:        Rook FlexVolume driver
+Group:          System/Filesystems
+
+%description rookflex
+Rook uses FlexVolume to integrate with Kubernetes for performing storage
+operations.
+
+################################################################################
+# Rook and Ceph manifests metadata
+################################################################################
+%package k8s-yaml
+Summary:        Kubernetes YAML file manifests for deploying a Ceph cluster
+Group:          System/Management
+BuildRequires:  ceph
+BuildArch:      noarch
+
+%description k8s-yaml
+This package contains examples of yaml files required to deploy and run the
+Rook-Ceph operator and Ceph clusters in a Kubernetes cluster.
+
+################################################################################
+# Rook ceph operator helm charts
+################################################################################
+%package ceph-helm-charts
+Summary:        Rook Ceph operator helm charts
+Group:          System/Management
+BuildArch:      noarch
+
+%description ceph-helm-charts
+Helm helps manage Kubernetes applications. Helm Charts define,
+install, and upgrade Kubernetes applications. Rook is a
+cloud-native storage orchestrator for Kubernetes, providing
+the platform, framework, and support for a diverse set of storage
+solutions to integrate with cloud-native environments.
+
+This package contains Helm Charts for Rook.
+
+################################################################################
+# Rook integration test binary metadata
```

Comment:
Do we still need this given we won't be running those tests now?

Reply:
Removed all references to these test binaries. Thanks for pointing that out.